angular-template.js

```diff
@@ -52,6 +70,13 @@ module.exports = {
 
         '@angular-eslint/template/no-distracting-elements': 'error',
 
-        '@angular-eslint/template/no-duplicate-attributes': 'error'
+        '@angular-eslint/template/no-duplicate-attributes': 'error',
+
+        /*
+            This rule primarily impacts accessibility so currently requires enforcement per project.
+        */
+        '@angular-eslint/template/no-positive-tabindex': 'off',
```

Comment:
Is this really the best strategy? I feel like a team that needs to enable the accessibility rules would prefer them grouped and outlined under the comment.

Reply:
It makes it easier to maintain to have the rules listed and ordered like they print in the tools. We could tag them in a comment like:

```
        /*
            [accessibility] This rule primarily impacts accessibility so currently requires enforcement per project.
        */
        '@angular-eslint/template/no-positive-tabindex': 'off',
```
To make that group of rules easier to identify. I like that idea for also marking rules pertaining to strict null checks, maybe a comment tag like `[strict-null-checks]`.

Reply:
Added rule groups for `[accessibility]`, `[strict-null-checks]`, `[application-prefix]`, and corresponding content to the README.